osbenchmark/workload/params.py

```diff
@@ -888,6 +888,7 @@ def __init__(self, workload, params, context: Context, **kwargs):
         self.data_set_format = parse_string_parameter("data_set_format", params)
         self.data_set_path = parse_string_parameter("data_set_path", params, "")
         self.data_set_corpus = parse_string_parameter("data_set_corpus", params, "")
+        self.data_set_name = params.get("data_set_name")
```

Comment:
should we use parse_string_parameter call as done above? Also, how does this relate to this change?

Reply:
To read parquet, we need the column name from users. Based on my observation, i see different column is used to store vectors in parquet file.